circle.yml

```diff
@@ -22,6 +22,7 @@ references:
           name: test
           command: |
             docker-compose run $CIRCLE_JOB bundle exec rake test
+            docker-compose run $CIRCLE_JOB RUBYOPT="--enable-frozen-string-literal" bundle exec rake test
```

Comment:
Trying to run the test, but it seems like Circle is not working...

Reply:
Thank you for letting me know, I'll migrate these test to GH Actions